apps/wing/project-templates/wing/react-vite/package.json

```diff
@@ -5,7 +5,9 @@
   "author": "Your Name",
   "license": "MIT",
   "scripts": {
-    "postinstall": "(cd backend && npm install) && (cd frontend && npm install)"
+    "install:backend": "cd backend && npm install",
+    "install:frontend": "cd frontend && npm install",
+    "postinstall": "npm run install:backend && npm run install:frontend"
```

Comment:
If possible our template should use npm workspaces

Reply:
tried it and it worked for this specific tempalte:
```
  "workspaces": ["backend", "frontend"],
```
will it work will other Wing apps? node_modules folder is created at the root folder and not under `backend`

Reply:
Sadly it might not, we don't really do any testing with that. Not a blocker for this PR but I did notice when testing this out that the initial install is super slow